### PR TITLE
feat(Portable): MessageBox implementation for NET46

### PR DESCRIPTION
**Pull request by GantMan**
This is a simplification of the modules/dialog.

* Shares constants in shared
* Implements basic MessageBox for .NET46

### CODE REVIEW

File: ReactWindows/ReactNative.Shared/Modules/Dialog/DialogModuleHelper.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared/Modules/Dialog/DialogModuleHelper.cs]: cleanup unused namespaces.

File: ReactWindows/ReactNative.Net46/Modules/Dialog/DialogModule.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using ReactNative.Collections;
+using System.Collections.Generic;
+using System.Windows;
+// using Windows.ApplicationModel.Core;
+// using Windows.UI.Core;
+// using Windows.UI.Popups;
```
Comment (rozele) [inline ReactWindows/ReactNative.Net46/Modules/Dialog/DialogModule.cs]: Delete commented out namespaces.

File: ReactWindows/ReactNative.Shared/Modules/Dialog/DialogModuleHelper.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+namespace ReactNative.Modules.Dialog
+{
+    static class DialogModuleHelper
```
Comment (GantMan) [inline ReactWindows/ReactNative.Shared/Modules/Dialog/DialogModuleHelper.cs]: I was hoping to leave room for eventual shared code here, perhaps shared methods.  Sort of a placeholding class for constants and helper methods.  I can extract them if you'd like.  Let me know what you prefer.

The readability seems good.

**Comment by GantMan** [comment]
@rozele - cleaned up the namespaces.  Just let me know if you want me to extract the shared back out.  Will do if that's what is decided 👍

File: ReactWindows/ReactNative.Net46/Modules/Dialog/DialogModule.cs [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+                if (result == MessageBoxResult.OK)
+                {
+                    actionCallback.Invoke(DialogModuleHelper.ActionButtonClicked, DialogModuleHelper.KeyButtonPositiveValue);
+                }
+                else
+                {
+                    actionCallback.Invoke(DialogModuleHelper.ActionButtonClicked, DialogModuleHelper.KeyButtonNegativeValue);
+                }
+            } else if (containsPositive)
+            {
```
Comment (rozele) [inline ReactWindows/ReactNative.Net46/Modules/Dialog/DialogModule.cs]: Looks like you're mixing K&R and Allman indentation here.

Comment (GantMan) [inline ReactWindows/ReactNative.Net46/Modules/Dialog/DialogModule.cs]: Hah, JavaScript has me mixed with The K&R.  Fixing back to Allman.

**Comment by rozele** [comment]
:shipit:

**Comment by GantMan** [comment]
gonna rebase once the other PR #759 is merged.

**Comment by rozele** [comment]
Rebase and good to go.

**Comment by GantMan** [comment]
@rozele 👍

**Pull request merged**